packages/lesspass-entropy/src/index.js

```diff
@@ -10,6 +11,22 @@ function calcEntropy(profile, masterPassword) {
   return pbkdf2(masterPassword, salt, iterations, keylen, digest);
 }
 
+function generateUserKey() {
```

Comment:
We need to add some unit test to valid those methods. If we implement this in a bad way, it can introduce some potential risk for the user